packages/language-service/src/plugins/vue-twoslash-queries.ts

```diff
@@ -1,7 +1,7 @@
 import { ServicePlugin, ServicePluginInstance } from '@volar/language-service';
 import * as vue from '@vue/language-core';
-import { Provide } from 'volar-service-typescript';
 import type * as vscode from 'vscode-languageserver-protocol';
+import { getQuickInfoAtPosition } from 'typescript-vue-plugin/out/namedPipe/client';
```

Comment:
Should it rely on internal package build paths?

Reply:
Not sure I understand. Just need to make sure that both node and esbuild can interpret the path.

Reply:
just saying that `/out/namedPipe/client` path is an internal detail of the `typescript-vue-plugin` package and should not be hardcoded like that. But since you control both, I guess you can get away with it :)